autoload/vimspector.vim

```diff
@@ -469,6 +469,20 @@ function! vimspector#JumpToBreakpointViewBreakpoint() abort
   py3 _vimspector_session.JumpToBreakpointViewBreakpoint()
 endfunction
 
+function! vimspector#JumpToNextBreakpoint() abort
```

Comment:
New issue: This file is almost entirely composed of such forwarding functions. They could be generated from a list instead.